Sites/styles.css

```diff
@@ -0,0 +1,80 @@
+body {
+    @media (max width: 1280px) @media (min-width: 375px) height: 4600px;
```

Comment:
This media query is broken, but you also probably don't need a height on your `body`, so it's safe to delete:
```suggestion
```
The proper way to write media queries would be :
```css
body {
    background-color: #ebf0fa;
}

@media (min-width:600px) {
    body {
        background: linear-gradient(#ebf0fa, #adc2eb);
    }
}
```
This CSS would set your `background-color` to be `#ebf0fa` initially and then change it to the gradient once the browser window was at least 600px wide. Note that you have to include the element you are styling inside the media query (in this case, the `body` element). You can also include multiple element inside one media query, e.g.:
```css
@media (min-width:600px) {
    body {
        background: linear-gradient(#ebf0fa, #adc2eb);
    }
    h1 {
        font-size: 60px;
    }
}
```
In the code above, not only does the background change starting at 600px, but the `font-size` of the `h1` would also be reset.